framework/packages/abstract-sdk/src/apis/grant.rs

```diff
@@ -59,16 +62,27 @@ impl Grant {
         grantee: &Addr,
         basic: BasicAllowance,
     ) -> AbstractSdkResult<AccountAction> {
-        let msg = feegrant::v1beta1::MsgGrantAllowance {
+        let allowance = feegrant::v1beta1::BasicAllowance {
+            spend_limit: vec![SdkCoin {
+                denom: "uosmo".to_string(),
```

Comment:
Guessing this was just for testing?

Reply:
Yes - pulled out the fee-grant changes into #41